From a75305af33d8113e0e2914c7f50e61d3f3f79746 Mon Sep 17 00:00:00 2001
From: Steffen Olszewski <steffen.olszewski@gero-mess.de>
Date: Wed, 13 Dec 2023 21:43:00 +0100
Subject: [PATCH] Prevent installation of TinyXML2 development files

Simply exclude the library from the ALL target for now.
If the library gets build as shared library, the runtime component
needs to be installed explicit.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4ef78e5a..7fdf2e1e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -104,7 +104,7 @@ add_subdirectory(sdk)
 add_subdirectory(third_party/md5)
 add_subdirectory(third_party/stack_trace)
 stash_variable(BUILD_SHARED_LIBS ${WXFB_TINYXML2_SHARED})
-add_subdirectory(third_party/tinyxml2)
+add_subdirectory(third_party/tinyxml2 EXCLUDE_FROM_ALL)
 restore_variable(BUILD_SHARED_LIBS)
 
 if(NOT WXFB_BUILD_PLUGIN_HOST)
